create new EC web ui screenshot .md file

vidya22022 opened the pull request:
created new EC WEB-UI screenshot .md file

vidya22022 committed:
create new EC web ui screenshot .md file

ayasuda-ge commented:
could you check-in the screenshots/images? @vidya22022 @vidya2205 @Subhash-vishwakarma

ayasuda-ge reviewed (changes requested):
- please check-in the screenshots.
- refer the screenshots in README.
@vidya22022 @vidya2205

vidya22022 committed:
Create screens

vidya22022 committed:
Add files via upload

vidya22022 committed:
Create test

created maintain folder

vidya22022 committed:
Add files via upload

added screenshot of maintain tabs

vidya22022 committed:
Create test

create folder for view screenshot

vidya22022 committed:
Add files via upload

added screnshot for view

vidya22022 committed:
Create test

created setting folder

vidya22022 committed:
Add files via upload

added screenshots for settings tab

vidya22022 committed:
Update EC-UI-screenshot.md

changed screenshots

vidya22022 committed:
Delete test

vidya22022 committed:
Delete screens

vidya22022 committed:
Delete test

vidya22022 committed:
Delete test

ayasuda-ge commented on docs/EC-UI-screenshot.md:
```diff
@@ -0,0 +1,148 @@
+
+1.1)View: Topology View Table
+
+![image](https://user-images.githubusercontent.com/76679290/109985830-30292280-7d2b-11eb-8dbc-78e256fed96b.png)
```
please update the image link to the relative path. 

https://stackoverflow.com/questions/10189356/how-to-add-screenshot-to-readmes-in-github-repository

ayasuda-ge replied:
@vidya22022 @Subhash-vishwakarma

ayasuda-ge replied:
*image links

ayasuda-ge reviewed (changes requested):
please view the comment for the update.

vidya22022 committed:
Update EC-UI-screenshot.md

added relative path for images to view

vidya22022 committed:
Update EC-UI-screenshot.md

added relative path for images to view

vidya22022 committed:
added relative path

added relative path for images to view

vidya22022 committed:
added relative path

added relative path

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
updated path for images

updated path for images

ayasuda-ge commented on docs/EC-UI-screenshot.md:
```diff
@@ -0,0 +1,135 @@
+
+## 2)Maintain:
+
+2.1)Maintain: Subscription ->Create
```
Please use proper markdown indenting.

ayasuda-ge replied:
@Subhash-vishwakarma please assist @vidya22022

Subhash-vishwakarma replied:
@ayasuda-ge Vidhya raised a new PR using proper markdown indenting. Please have a review and let us know if any further changes required

vidya22022 committed:
Updated images names

Updated images names

vidya22022 committed:
added list of content in document

added list of content in document

vidya22022 committed:
added headlines structure

added headlines structure

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

added sub content for view

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
added index as per previous comments.

added index with links as per previous comments.

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

done some changes in code

vidya22022 committed:
Update EC-UI-screenshot.md

added some changes for index

vidya22022 committed:
Delete ViewTreelist.png

vidya22022 committed:
Add files via upload

vidya22022 committed:
Create WebUI_Kt_Document

created new file for KT  document

vidya22022 committed:
Update EC-UI-screenshot.md

added numbers for inexing

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update and rename WebUI_Kt_Document to WebUI_Kt_Document.md

created document for some web Ui kt

vidya22022 committed:
Create test

vidya22022 committed:
Delete test

vidya22022 committed:
Create test

vidya22022 committed:
Add files via upload

vidya22022 committed:
Delete test

vidya22022 committed:
Update WebUI_Kt_Document.md

vidya22022 committed:
Update WebUI_Kt_Document.md

vidya22022 committed:
Update WebUI_Kt_Document.md

vidya22022 commented:
added new screenshot for topology and update index @ayasuda2OO3

ayasuda-ge commented on docs/EC-UI-screenshot.md:
```diff
@@ -0,0 +1,142 @@
+# EC WEB-UI Screenshots as per Tabs
+ * [1.View](#1_view)
```
please use proper markdown numeric indent example here-

https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list

@vidya22022 @Subhash-vishwakarma

Subhash-vishwakarma replied:
yeah Sure @ayasuda-ge will try to close this today

vidya22022 replied:
using numeric markdown we got this solution only else we can use bullets markdown @ayasuda-ge

Subhash-vishwakarma replied:
@ayasuda-ge  By this solution we mean using of &nbsp; for space there / for new line

ayasuda-ge replied:
Consider this @Subhash-vishwakarma @vidya22022

1. this is the proper mardown
  1.1 another market down
  1.2 next mark down
  1.3 blah blah
2. final markdown
3. summary

vidya22022 replied:
@ayasuda-ge  We marked index with above format but in raw we have used & &nbsp for space we also tried with \ but its not working so is that fine to use &nbsp? or we can go with bullets also one more question i have i created 1 KT document in different file do you want me to add on same screenshot file or its keep as it is separate ?

vidya22022 committed:
Update EC-UI-screenshot.md

remove bulletes

vidya22022 committed:
Update EC-UI-screenshot.md

added font

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update EC-UI-screenshot.md

for using numerical markdown we get only above solution else we can use bullets markdown indentation

vidya22022 committed:
Update EC-UI-screenshot.md

vidya22022 committed:
Update WebUI_Kt_Document.md

added some content in document

vidya22022 committed:
Update WebUI_Kt_Document.md

vidya22022 committed:
Update WebUI_Kt_Document.md

ayasuda-ge reviewed (approved):
Thanks for the contribution. @vidya22022 @Subhash-vishwakarma
